tests/integration-tests/tests/common/assertions.py

```diff
@@ -63,9 +63,19 @@ def assert_scaling_worked(
         + minutes(estimated_scaleup_time)
         + minutes(max_scaledown_time),
     )
-    if assert_asg:
-        assert_that(max(asg_capacity_time_series)).is_equal_to(expected_max)
-        assert_that(asg_capacity_time_series[-1]).is_equal_to(expected_final)
-    if assert_scheduler:
-        assert_that(max(compute_nodes_time_series)).is_equal_to(expected_max)
-        assert_that(compute_nodes_time_series[-1]).is_equal_to(expected_final)
+
+    with soft_assertions():
+        if assert_asg:
+            assert_that(max(asg_capacity_time_series)).described_as(
+                f"asg_capacity_time_series={asg_capacity_time_series}"
```

Comment:
Is it right you are using the same description for the two expected_max and expected_final comparison?

If yes we could set a variable for the description (`f"asg_capacity_time_series={asg_capacity_time_series}"`) and then use it for the two asserts. The same below for `f"compute_nodes_time_series={compute_nodes_time_series}"`